inference-engine/tests/functional/shared_test_classes/src/single_layer/eltwise.cpp

```diff
@@ -52,25 +60,18 @@ InferenceEngine::Blob::Ptr EltwiseLayerTest::GenerateInput(const InferenceEngine
 }
 
 void EltwiseLayerTest::SetUp() {
-    std::vector<std::vector<size_t>> inputShapes;
+    std::pair<std::vector<ngraph::PartialShape>, std::vector<std::vector<ngraph::Shape>>> shapes;
     InferenceEngine::Precision netPrecision;
     ngraph::helpers::InputLayerType secondaryInputType;
     CommonTestUtils::OpType opType;
     ngraph::helpers::EltwiseTypes eltwiseType;
     std::map<std::string, std::string> additional_config;
-    std::tie(inputShapes, eltwiseType, secondaryInputType, opType, netPrecision, inPrc, outPrc, inLayout, targetDevice, additional_config) =
+    std::tie(shapes, eltwiseType, secondaryInputType, opType, netPrecision, inPrc, outPrc, inLayout, targetDevice, additional_config) =
         this->GetParam();
     auto ngPrc = FuncTestUtils::PrecisionUtils::convertIE2nGraphPrc(netPrecision);
+    std::tie(inputDynamicShapes, targetStaticShapes) = shapes;
 
-    std::vector<size_t> inputShape1, inputShape2;
-    if (inputShapes.size() == 1) {
-        inputShape1 = inputShape2 = inputShapes.front();
-    } else if (inputShapes.size() == 2) {
-        inputShape1 = inputShapes.front();
-        inputShape2 = inputShapes.back();
-    } else {
-        IE_THROW() << "Incorrect number of input shapes";
-    }
+    ngraph::Shape inputShape1 = targetStaticShapes.front().front(), inputShape2 = targetStaticShapes.front().back();
 
     configuration.insert(additional_config.begin(), additional_config.end());
     auto input = ngraph::builder::makeParams(ngPrc, {inputShape1});
```

Comment:
I guess we also need to `set_friendly_name` for params to have correct order when we iterate through input map

Reply:
fixed